api-docs/location-api-intro.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Location API - Introduction
+category: ???
```

Comment:
The Category id for this is `6453af734f393c01aee97223`

Reply:
Resolved! Thanks for finding the ID 🙌🏻